weather-app/src/components/Footer/WeatherList/index.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+
+import { useTypedSelector } from 'hooks';
+import Today from './Today';
+import WeatherListItem from './WeatherListItem';
+
+import { StyledWeatherList, StyledWeatherListContainer } from './styles';
+
+const WeatherList: React.FC = () => {
+  const { weatherData, weatherLoading, weatherStateError } = useTypedSelector(
+    (state) => state.weatherState,
+  );
+  if (weatherLoading) return <div>loading</div>;
+  if (weatherStateError) return <div>error</div>;
```

Comment:
To improve performance, instead of `if` recommended to use the operators `?` and `&&` in `return`